### ISSUE TITLE
reconstruct_from_listmode crashes

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/SIRF#143]
both Matlab and Python. seems to be when creating the attenuation correction factors again.

**Comment by KrisThielemans** [comment]
running with STIR and SIRF compiled in debug mode gives
```
applying attenuation (please wait, may take a while)...
python: /usr/include/boost/smart_ptr/shared_ptr.hpp:648: typename boost::detail::sp_member_access<T>::type boost::shared_ptr<T>::operator->() const [with T = stir::BackProjectorByBin; typename boost::detail::sp_member_access<T>::type = stir::BackProjectorByBin*]: Assertion `px != 0' failed.
```
saying that the backprojector ptr is set to zero.

that's weird though as the attenuation has no reason to have a backprojector. not sure where this is happening.

**Comment by KrisThielemans** [comment]
This was caused because I deleted an `acq_model.set_up()` in 8c304627b66d63a82fa117a46c242c7ba6eb6fa9. However, this brought some problems to light as we should never crash of course. Since merging #144, SIRF will throw an error as opposed to crashing. In addition, UCL/STIR@73e42520331473a02a3b983a1cf8c4a1b247a6e7 fixed the underlying cause.

**Comment by KrisThielemans** [comment]
fixed with 4948ccd4dbe13d69622486c2372a07a815535c00 and recent STIR fixes
